.github/in-cluster-test-scripts/multicluster.sh

```diff
@@ -56,4 +56,4 @@ sleep 10s
 [[ $(pgrep -f "cilium.*hubble.*port-forward|kubectl.*port-forward.*hubble-relay" | wc -l) == 2 ]]
 
 # Run connectivity test
-cilium --context "${CONTEXT1}" connectivity test --debug --multi-cluster "${CONTEXT2}" --test '!/*-deny,!/pod-to-.*-nodeport' --all-flows
+cilium --context "${CONTEXT1}" connectivity test --debug --multi-cluster "${CONTEXT2}" --test '!cluster-entity*,!/*-deny,!/pod-to-.*-nodeport' --all-flows
```

Comment:
Same here, could you document the reasons?

I'm not sure why we're skipping some of those other ones either, but it makes me wonder about them. I would have thought that multi-cluster should also test cluster entity to confirm the behaviour between clusters. Once we've added these tests into the CLI, any users may start running these tests, including the ones that we are skipping in CI. They won't have the benefit of the github action telling them which tests are good and which ones are bad.

Reply:
not sure about other tests, but for cluster entity, it's mainly due to below the particular setting, which is to restrict only local cluster label.

https://github.com/cilium/cilium/blob/master/pkg/policy/api/entity.go#L126

Reply:
I have adjusted the test, so that we only verify for local node.